src/js/vertical-slide.js

```diff
@@ -1,49 +1,48 @@
 // Variables
+const hostName = location.hostname;
```

Comment:
This is where [`wp_localize_script`](https://developer.wordpress.org/reference/functions/wp_localize_script/) really comes in handy. You can't depend on the wp-json location being at the hostname root, so getting to the script as a variable is the trick to this one. You might add an object that has the full rest route as a property, using [`get_rest_url`](https://developer.wordpress.org/reference/functions/get_rest_url/) to retrieve the correct URL.

Reply:
I should define the problem in a different branch.